1_Explore_a_Prediction.py

```diff
@@ -11,6 +11,7 @@
 
 # Sidebar ------------------------------------
 filtering.view_selection()
+filtering.view_model_select()
```

Comment:
Personally I think this selectbox would make more sense being the first select box - right now, it kind of gets buried on pages with more dropboxes (like change over time)